packages/utils/src/CypressUtils/TableUtils.js

```diff
@@ -0,0 +1,41 @@
+import { Cypress, cy } from 'cypress';
+import _ from 'lodash';
+
+import { ROW, TABLE, TBODY, TITLE } from './selectors';
+
+export function checkTableHeaders(expectedHeaders) {
+  /* patternfly/react-table-4.71.16, for some reason, renders extra empty `th` container;
```

Comment:
The empty th is for the column with checkboxes. It should be only the case for tables with selectable rows

Reply:
So, should I add a parameter to the function that would indicate if the rows are selectable or not? That way we could utilize this test for both types of tables